Makefile.am

```diff
@@ -40,6 +40,13 @@ else
 	PARANEURON='no'
 endif
 
+# Generate help_data.dat since it's no longer committed in the repo.
+.PHONY: help_data_dat
+help_data_dat:
+	$(PYTHON_BLD) $(top_srcdir)/docs/parse_rst.py $(top_srcdir)/docs/python $(top_srcdir)/share/lib/python/neuron/help_data.dat
+
+-include help_data_dat
+
```

Comment:
If this works, fine. But I don't mind if the autotools gets less and less complete.

Reply:
I tried to have a working solution (still autotools rookie 😄 ) . I do believe it's appropriate.

Reply:
@nrnhines If autotools doesn't handle help_data correctly, then some of the CI tasks should fail.

Reply:
Indeed, this is also run for autotools: https://github.com/neuronsimulator/nrn/blob/master/share/lib/python/neuron/tests/test_neuron.py#L175